Sources/OTT/Analytics/BaseOTTAnalyticsPlugin.swift

```diff
@@ -29,6 +29,8 @@ public class BaseOTTAnalyticsPlugin: BasePlugin, OTTAnalyticsPluginProtocol, App
     
     var disableMediaHit: Bool = false
     var disableMediaMark: Bool = false
+    var isExperimentalLiveMediaHit: Bool = false
+    var isLiveMedia: Bool = false
```

Comment:
Is there a reason for this?
There is an isLive function on the player, can that be used?